test/suite/test_backup15.py

```diff
@@ -126,13 +143,17 @@ def take_full_backup(self):
 
                 shutil.copy(copy_from, copy_to)
         self.assertEqual(ret, wiredtiger.WT_NOTFOUND)
-        cursor.close()
+        bkup_c.close()
 
     def take_incr_backup(self):
         # Open the backup data source for incremental backup.
         buf = 'incremental=(src_id="ID' +  str(self.counter-1) + '",this_id="ID' + str(self.counter) + '")'
```

Comment:
The previous commit added this statement here:
```
 self.assertTrue(self.counter > 0)
```
I think that's helpful to have.  Alternatively it could be put at the top of `range_copy`, where `self.counter` is used.

Reply:
Fixed. Thanks for catching that I agree it is helpful to have.